 <launch>
   <node pkg="tools" type="mux.py" name="mux">
      <remap from="in_A" to="/control/speed"/>
      <remap from="in_B" to="/vision/speed_F"/>
      <remap from="in_C" to="/vision/speed_OA">
      <remap from="in_Sel" to="/control/joy"/>
      <remap from="out" to="/motor/speed"/>
   </node>
   <!--
   <node pkg="rosserial_python" type="serial_node.py" name="rosserial"/>
   -->
</launch>
